packages/react/src/components/ProgressIndicator/ProgressIndicator.js

```diff
@@ -117,13 +119,18 @@ export function ProgressStep({
           invalid={invalid}
           prefix={prefix}
         />
-        <ProgressStepLabel className={`${prefix}--progress-label`}>
+        <ProgressStepLabel
+          className={`${prefix}--progress-label ${spaceEqually &&
+            prefix + '--progress-label--no-max-width'}`}>
```

Comment:
If we use a global modifier, we can avoid having to attach classNames to each element that needs to be updated, and don't have to track the prop to each individual element